grafana/data_source_organization_users.go

```diff
@@ -0,0 +1,62 @@
+package grafana
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DatasourceOrganizationUsers() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+* [Official documentation](https://grafana.com/docs/grafana/latest/administration/manage-organizations/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/org/)
+`,
+		ReadContext: dataSourceOrganizationUsersRead,
+		Schema: map[string]*schema.Schema{
+			"organization_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the Organization.",
+			},
+			"emails": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Description: "A list of the Organization member users' email addresses.",
+			},
```

Comment:
I considered enhancing the data source with a `users` property:

```golang
"users": {
  Type:     schema.TypeSet,
  Computed: true,
  Elem: &schema.Resource{
    Schema: map[string]*schema.Schema{
      "email": {
        Type:     schema.TypeString,
        Computed: true,
      },
      "login": {
        Type:     schema.TypeString,
        Computed: true,
      },
      "role": {
        Type:     schema.TypeString,
        Computed: true,
      },
      "id": {
        Type:     schema.TypeInt,
        Computed: true,
      },
    },
  },
```

Would others like to see that as part of this PR now? If so, should that replace or coincide with the existing `emails`?